src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/AddressPhoneRepository.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.repository;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.AddressPhone;
+
+import java.util.List;
+
+public interface AddressPhoneRepository extends CrudRepository<AddressPhone, Long> {
```

Comment:
This could do with its own test class, but not massively important as it is indirectly tested anyway.